lib/class-wp-rest-image-editor-controller.php

```diff
@@ -44,22 +44,22 @@ public function register_routes() {
 					'permission_callback' => array( $this, 'permission_callback' ),
 					'args'                => array(
 						'x'        => array(
-							'type'     => 'float',
+							'type'     => 'number',
 							'minimum'  => 0,
```

Comment:
Is it also possible to specify a maximum?

Reply:
Yep we can use the `maximum` property.